packages/icons/src/svg/solid/arrow-down-square.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I noticed the icons have various font sizes. Can we make sure they are all in 24x24. Please refer the icon from the design - https://www.figma.com/file/RHGjzR2s9b4v1L20T8WCg1/Mochi---Design-System?type=design&node-id=1-144&mode=design&t=XhB1ht8nXzEXsslm-4. If it's not 24x24, please ask the designer to update it.